documentation/src/main/asciidoc/index.asciidoc

```diff
@@ -4,54 +4,43 @@ Written and maintained by the {brandname} community
 
 Get answers and find help in the {brandname} {infinispanversion} titles.
 
-Getting Started::
+Embedding Infinispan::
 
-* link:titles/overview/overview.html[Technical Overview]
-* link:titles/getting_started/getting_started.html[Getting Started Guide]
```

Comment:
@tristantarrant I'm not sure about the overview and getting started docs. they seem pretty stale but might still be useful. should we keep them?